server/channels/db/migrations/postgres/000110_update_vacuuming.up.sql

```diff
@@ -0,0 +1,4 @@
+alter table posts set (autovacuum_vacuum_scale_factor = 0.1, autovacuum_analyze_scale_factor = 0.05);
+alter table threadmemberships set (autovacuum_vacuum_scale_factor = 0.1, autovacuum_analyze_scale_factor = 0.05);
+alter table fileinfo set (autovacuum_vacuum_scale_factor = 0.1, autovacuum_analyze_scale_factor = 0.05);
+alter table preferences set (autovacuum_vacuum_scale_factor = 0.1, autovacuum_analyze_scale_factor = 0.05);
```

Comment:
Tangential but is it common practice to define these vs something in the db config? I am wondering if these values are "one size fits all" or there could be cases where they'd need some custom tuning.

Also, do we know what's the cost (if any) of running these?

Reply:
> Tangential but is it common practice to define these vs something in the db config? 

You cannot specify these in the DB. These are table level tunings and need to be done via SQL queries. 

>  I am wondering if these values are "one size fits all" or there could be cases where they'd need some custom tuning.

For small dbs, they anyways don't matter much. It only makes a difference when your data set exceeds a certain threshold. So given our workload and the fact that we know these tables tend to be larger than the others, this is a safe optimization applicable to everyone.

> Also, do we know what's the cost (if any) of running these?  

No cost. It's instantenous.